src/main/java/duckling/Configuration.java

```diff
@@ -16,6 +16,7 @@
     public static final int DEFAULT_PORT = 5000;
     public static final String DEFAULT_ROOT = ".";
     public static final RouteDefinitions DEFAULT_ROUTES = new RouteDefinitions(
+        Routes.get("/redirect").andRedirectTo("http://localhost:5000/"),
```

Comment:
Having the full host/port combination in here seems a bit fragile: what if you want to bind the server to 0.0.0.0 and send internet traffic to it? Like if you hosted the cob_spec server itself on Heroku or something. 

I suggest just redirecting using the path: `/`. This would mean you don't have to change this code every time the server location changes (and it *should* still pass the cob_spec test).